src/server/controllers/s3.js

```diff
@@ -0,0 +1,44 @@
+const router = require('express').Router
+const ms = require('ms')
+const S3 = require('aws-sdk/clients/s3')
+
+const defaultConfig = {
+  acl: 'public-read',
+  key: process.env.UPPYSERVER_AWS_KEY,
+  secret: process.env.UPPYSERVER_AWS_SECRET,
+  bucket: process.env.UPPYSERVER_AWS_BUCKET,
+  region: process.env.UPPYSERVER_AWS_REGION,
+  conditions: []
+}
+
+module.exports = function s3 (config) {
+  config = Object.assign({}, defaultConfig, config)
+  const client = new S3({
```

Comment:
Hmm, this thing could perhaps be passed in by the user as well if they're using the `pluggable` way.